CONTRIBUTING.md

```diff
@@ -46,6 +46,8 @@ contributors.
   needed.
 - Add a `replace` directive at the root `go.mod` file so your component is
   included in the build of the contrib executable.
+- Once a new component has been added to the executable, please add the 
+  component to the [OpenTelemetry.io registry](https://github.com/open-telemetry/opentelemetry.io#adding-a-project-to-the-opentelemetry-registry).
```

Comment:
I think we somewhere refer to the core contribution section, maybe look where we do that and clarify that contrib should follow the same process.

Reply:
I'm not sure I understand, @bogdandrutu. Are you saying to refer to https://github.com/open-telemetry/opentelemetry-collector/pull/2550/files? If so, I can absolutely do that, but I believe there's a larger question around the duplicate CONTRIBUTING sections then. There's overlap in the collector and collector-contrib CONTRIBUTING sections, but this exists due to the differences in the types of contributions and to make it easy for new contributors to find where to start. In the case of this PR, if we link to the contrib repo we would have contrib-CONTRIBUTIONS -> core-CONTRIBUTIONS -> opentelemetry.io.

Reply:
I think we already refer to that here https://github.com/open-telemetry/opentelemetry-collector-contrib#prs-and-reviews

Reply:
After re-reviewing the section you referred to, @bogdandrutu, it may not be necessary to update anything, since this was already documented in the core collector. I now believe we can just close this PR...

Reply:
Yes, maybe if you re-read a clarification may be helpful for others. People may read as you did initially only this section.